data-es6.js

```diff
@@ -11808,6 +11808,10 @@ exports.tests = [
         try {
           Date.prototype.valueOf(); return false;
         } catch(e) {}
+        if (Object.prototype.toString.call(Error.prototype)  !== '[object Object]')
+          return false;
+        if (Object.prototype.toString.call(TypeError.prototype) !== '[object Object]')
```

Comment:
If you're going to check `TypeError`, what about all the other built in error types?